### DISCUSSION TITLE
Posterl look White in place of orignal image

### DISCUSSION THREAD

**Discussion by ti-hardikshah** [google/model-viewer#3015] [Q&A] [answered]
### Description

When Loading loading ="auto" and reveal="interaction" 
then the poster image does not look exactly image it look a little white, is there any way to update CSS ? 
Maybe it changes from the JS file. 
 
#### Live Demo
https://volcano-bouncy-front.glitch.me

#### Source
https://glitch.com/edit/#!/volcano-bouncy-front

### Version

- model-viewer: All

### Browser Affected

- [X] Chrome, version: xx.x.xxxx.xx
- [X] Edge
- [X] Firefox
- [X] IE
- [X] Safari

### OS

- [X] Android
- [X] iOS
- [ ] Linux
- [X] MacOS
- [X] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
Indeed, this is because of the progress mask, which I'm considering removing entirely. For now you can disable it by targeting your CSS to https://modelviewer.dev/docs/#entrydocs-loading-parts-defaultProgressMask

You can  see an example by  viewing source on: https://modelviewer.dev/examples/lighthouse2.html

**Comment by ti-hardikshah** [comment, marked as answer]
Thank you very much. 

i just need to update CSS which effect on poster 

`<style>
      model-viewer::part(default-progress-mask) {
      display: none;
    }
    </style>`
    
    And this resolves my issue.
